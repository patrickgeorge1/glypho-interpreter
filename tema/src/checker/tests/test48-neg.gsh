i---1--oo
