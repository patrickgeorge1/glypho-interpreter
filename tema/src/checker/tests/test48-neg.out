1
210345872
